src/mayur/assignment23/PrimeNumberOperations.java

```diff
@@ -0,0 +1,79 @@
+package mayur.assignment23;
+
+import java.util.Scanner;
+
+public class PrimeNumberOperations {
+	static int primeIndex = 0;
+	int[] getPrimeNumbers(int startIndex, int endIndex) {
+		int count = 0;
+		
+		int[] primeNumArr = new int[((endIndex - startIndex) / 2)];
+		for (int index = startIndex; index < endIndex; index++) {
```

Comment:
Take reference of this
Ankit - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1065/files